### DISCUSSION TITLE
Creating a Lollipop Plot with plotnine

### DISCUSSION THREAD

**Discussion by tongakuot** [has2k1/plotnine#839] [2024 Plotnine Contest]
### Authors

Alier Reng

### Links

https://www.linkedin.com/in/tongakuot/, https://www.youtube.com/@NileQuantumInsights, https://github.com/tongakuot

### Full description

This tutorial aims to educate Python users on how to create a segment or lollipop plot using the plotnine package. Similar to a barplot, the lollipop plot uses lines and dots to represent categories and their counts, providing a clear and concise visual representation of the data.
![lollipop-img-1](https://github.com/user-attachments/assets/25c8e3f2-0b9a-4a3c-b2a2-c46cb72ea4e3)

### Code repository

https://github.com/tongakuot/dataviz-plotnine/blob/main/segment-plot.ipynb

**Comment by has2k1** [comment]
![lolipop](https://github.com/user-attachments/assets/230b0116-c785-4bf0-8db2-54a8cd1e0fe4)

**Comment by EconomiCurtis** [comment]
Ahoy @tongakuot , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@tongakuot, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
